source/_integrations/zwave_js.markdown

```diff
@@ -90,12 +90,40 @@ Use this My button:
 1. Wait for the Z-Wave JS add-on to start up.
 1. Once the installation is complete, the **Device info** of the Z-Wave controller is shown.
    * You successfully installed the Z-Wave integration and the Z-Wave JS add-on.
-   * You can now add devices to the Z-Wave network.
+   * You can now [add](/integrations/zwave_js/#adding-a-new-device-to-the-z-wave-network) devices to the Z-Wave network.
 
 <p class='note'>
 While your Z-Wave mesh is permanently stored on your dongle, the additional metadata is not. When the Z-Wave integration starts up the first time, it will interview your entire Z-Wave network. Depending on the number of devices paired with the Z-Wave dongle, this can take a while. You can speed up this process by manually waking up your battery-powered devices. Most of the time, this is a button press on those devices (see their manual). It is not necessary to exclude and re-include devices from the mesh.
 </p>
 
+### Adding a new device to the Z-Wave network
+
+1. In Home Assistant, go to {% my integrations title="**Settings** > **Devices & Services**" %}.
+1. In the Z-Wave integration, select **Configure**.
+1. Select **Add device**.
+   * The Z-Wave controller is now in inclusion mode.
+1. If your device supports SmartStart, select **Scan QR code** and scan the QR code on your device.
+1. If your device does not support SmartStart, set the device in inclusion mode. Refer to the device manual to see how this is done.
+   * If your device is included using S2 security, you may be prompted to enter a PIN number provided with your device. Often, this PIN is provided with the documentation _and_ is also printed on the device itself. For more information on secure inclusion, refer to [this section](/integrations/zwave_js/#should-i-use-secure-inclusion).
+1. The UI should confirm that the node was added. After a short while (seconds to minutes), the entities should also be created.
+1. If the controller fails to add/find your device, cancel the inclusion process.
+   * In some cases, it might help to first [remove](/integrations/zwave_js/#removing-a-device-from-the-z-wave-network) a node (exclusion) before you add it, even when the device has not been added to this Z-Wave network yet.
```

Comment:
```suggestion
   * In some cases, it might help to first [remove](/integrations/zwave_js/#removing-a-device-from-the-z-wave-network) a device (exclusion) before you add it, even when the device has not been added to this Z-Wave network yet.
```